.github/workflows/test.yml

```diff
@@ -3,6 +3,11 @@ on:
   pull_request:
     types: [opened, synchronize]
 
+env:
+  AWS_WEB_IDENTITY_TOKEN_FILE: /tmp/awscreds
```

Comment:
Skimming some [AWS documentation around this subject](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/) it looks like a path like `/var/run/secrets/eks.amazonaws.com/[role name]/token` might be more idiomatic. What do you think?

Reply:
The directory `/var/run/secrets/eks.amazonaws.com/[role name]/` need to exists first before we can store the token there no? Otherwise we will just get a `No such file or directory` error message?

Reply:
I don't know what CDK does with that path, but it's possible it'll create the directory before attempting to create the file. If so, changing this should just work™. If not, we could either create the directory ourselves or use a more obvious `/tmp` filename based on that path, like `/tmp/aws-eks-[role-name]-token`.

Reply:
Gave it a go in [b06747e](https://github.com/linz/geostore/commit/b06747ed9cf31715217c80cc611d10adffd9f3c7) and it didn't work. Have opted for a more obvious filename in tmp. Thought about creating the path but my preference would be to keep the workflow steps simple. I see the PR has been approved, though I am more than happy to revisit and tidy up later.